index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
```suggestion
<html lang="ja">
```
I think it's better to change this part.

Reply:
Thank you for pointing this out.
We will change it.